packages/account/src/Types/common-prop.type.ts

```diff
@@ -94,3 +81,23 @@ export type TBinaryRoutes = {
     is_logged_in: boolean;
     is_logging_in: boolean;
 };
+
+export type TPersonalDetailsForm = {
```

Comment:
@yauheni-deriv This is not used anywhere, Is it? 🤔

Reply:
Will remove this. It was part of a migration